src/js/common/schemaform/definitions/dateRange.js

```diff
@@ -17,7 +17,7 @@ export function uiSchema(from = 'From', to = 'To', rangeError = 'To date must be
       validateDateRange
     ],
     'ui:errorMessages': {
-      dateRange: rangeError,
+      pattern: rangeError,
```

Comment:
Just following suit with all the rest of the 'ui:errorMessages' properties in our ui schema definitions (date, ssn, bankAccount, currentOrPastDate). No reason other than consistency.